app/src/main/java/com/dessalines/thumbkey/keyboards/ENThumbKeyProgrammerWide.kt

```diff
@@ -21,6 +22,76 @@ import com.dessalines.thumbkey.utils.SwipeDirection
 import com.dessalines.thumbkey.utils.SwipeNWay
 import com.dessalines.thumbkey.utils.autoCapitalizeI
 
+val SPACEBAR_ALL_SYMBOLS = SPACEBAR_TYPESPLIT_TOP_KEY_ITEM.copy(
```

Comment:
Move these to commonkeys.kt, and also add them to the FIThumbkeyWide.kt keyboard if you would.